dkg/frost/frost.go

```diff
@@ -24,6 +24,10 @@ func init() {
 	types.InitBLS()
 }
 
+// FROST contains network to broadcast messages, signer to sign outgoing messages,
+// storage to store keygen output, config to store protocol configuration such as id,
+// threshold, operator list etc and state contains properties that changes over the
+// runtime of the protocol like current round, msgs etc
 type FROST struct {
```

Comment:
Can we follow same style as:
https://github.com/RockX-SG/ssv-spec/blob/dkg/qbft/instance.go#L16-L25
i.e. Name it Instance, Wrap Network, Signer etc in IConfig?